app/controllers/practices_controller.rb

```diff
@@ -189,32 +209,17 @@ def update
       if is_cropping?(params[:practice])
         @practice.main_display_image.reprocess!
       end
-
-      partner_keys.each do |key|
-        next if @practice.practice_partners.ids.include? key.to_i
-
-        @practice.practice_partner_practices.create practice_partner_id: key.to_i
-      end
-
-      dept_keys = []
-      dept_keys = params[:practice][:department].keys if params[:practice][:department].present?
-      @practice.department_practices.each do |department|
-        department.destroy unless dept_keys.include? department.department_id.to_s
-      end
-      dept_keys.each do |key|
-        next if @practice.departments.ids.include? key.to_i
-
-        @practice.department_practices.create department_id: key.to_i
-      end
     end
 
     respond_to do |format|
       if updated
-        format.html { redirect_back fallback_location: root_path, notice: 'Practice was successfully updated.' }
-        format.json { render :show, status: :ok, location: @practice }
-      else
-        format.html { redirect_back fallback_location: root_path, alert: 'Practice could not be updated.' }
-        format.json { render json: @practice.errors, status: :unprocessable_entity }
+        if params[:next]
+          format.html { redirect_to "/practices/#{@practice.slug}/edit/#{params[:next]}", notice: 'Practice was successfully updated.' }
```

Comment:
we're getting a security ding here.
i'm thinking we refactor this to use a hash that has {'from-slug': 'to-next-slug', etc... } instead of using a param from the client.